frontend/src/views/assignments/offer-actions.js

```diff
@@ -58,11 +59,29 @@ function OfferActionButtons(props) {
         setOfferForAssignmentRejected,
     } = props;
 
+    const [show, setShow] = React.useState(false);
+
+    const hideOfferWithdrawConfirmation = () => setShow(false);
+    const showOfferWithdrawConfirmation = () => setShow(true);
+
     function createOffers() {
         for (const assignment of selectedAssignments) {
             offerForAssignmentCreate(assignment);
         }
     }
+    function confirmOfferWithdraw() {
+        // if withdrawing multiple offers at once, show confirmation
+        if (selectedAssignments && selectedAssignments.length > 1) {
```

Comment:
Will `selectedAssignments` ever not be a list? If it can be `null`, use `selectedAssignments?.length > 1`